### PR TITLE
KOGITO-4944 - Skip kogito-apps UI build

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -129,6 +126,8 @@ pipeline {
             steps {
                 script {
                     getMavenCommand('kogito-apps', true, true)
+                        .withProperty('skip.ui.build')
+                        .withProperty('skip.ui.deps')
```
Comment (ricardozanini) [inline Jenkinsfile:130]: We already have these defined in kogito-apps pom, I suppose?

File: Jenkinsfile [hunk 2]
```diff
@@ -110,10 +110,7 @@ pipeline {
                 script {
                     // Skip unnecessary plugins to save time.
                     getMavenCommand('optaplanner', true, true)
-                        .withProperty('enforcer.skip')
-                        .withProperty('formatter.skip')
-                        .withProperty('impsort.skip')
-                        .withProperty('revapi.skip')
+                        .withProperty('quickly')
```
Comment (ricardozanini) [inline Jenkinsfile:113]: Depends on #1204

**Comment by ricardozanini** [comment]
Depends on #1204

File: Jenkinsfile (hunk 2 above)
Comment (radtriste) [reply Jenkinsfile:113]: nope this is for Optaplanner ;)

File: Jenkinsfile (hunk 1 above)
Comment (radtriste) [reply Jenkinsfile:130]: like suggested in chat, we should a specific profile in kogito-apps which would be activated with `skipUI` property so we can do, even locally, `mvn clean install -DskipUI`

**Comment by radtriste** [comment]
> Depends on #1204

There is no dependency yet on that PR as the `quickly` property is for Optaplanner not for kogito-runtimes

I will update for kogito-runtimes in https://github.com/kiegroup/kogito-runtimes/pull/1225 once #1204 is merged

File: Jenkinsfile (hunk 2 above)
Comment (radtriste) [inline Jenkinsfile:113]: we are in runtimes so we want to test Optaplanner... `quickly` does not run tests in fact ... see https://github.com/kiegroup/optaplanner/blob/master/build/optaplanner-build-parent/pom.xml#L1158

Comment (cristianonicolai) [reply Jenkinsfile:113]: :/ ok, will rollback that

File: Jenkinsfile (hunk 1 above)
Comment (cristianonicolai) [reply Jenkinsfile:133]: will use properties for now as we have other issues defining the profile in kogito-apps

**Review by radtriste — APPROVED** [review]
Could we have something similar in kogito-examples ?

**Comment by cristianonicolai** [comment]
> Could we have something similar in kogito-examples ?

@radtriste seems like we dont build kogito-apps anymore when building kogito-examples

**Comment by radtriste** [comment]
> > Could we have something similar in kogito-examples ?
> 
> @radtriste seems like we dont build kogito-apps anymore when building kogito-examples

yeah right ... my bad. Forgot now apps and examples are decorrelated

**Review by ricardozanini — APPROVED** [review]

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Pull request merged**